### ISSUE TITLE
Secure master client communication

### ISSUE THREAD

**Issue by fl4p** [jhuckaby/Cronicle#497]
It appears that master client communication is not encrypted, even with https enabled.
Although it is impossible for attackers to authenticate, they might be able to read sensitive data stored in scripts.

This might be an edge case, but I think modern applications should use https instead of http when ever possible. This can easily work with self-signed certificates (using an https.Agent with `rejectUnauthorized: false`)

**Comment by jhuckaby** [comment]
Yup, this is a known issue / limitation with Cronicle v0.  HTTPS/SSL isn't really supported, and especially not at the websocket (server-to-server) level.  The whole system needs to be ripped out and redesigned, which is happening with Cronicle v2 (Orchestra).  In v2 everything can be SSL, including all server websocket connections, and the UI.

Cronicle v2 (Orchestra) will be released this year (2022).
